includes/api/coupon-get.php

```diff
@@ -67,6 +67,14 @@ function getCouponList($request)
         'posts_per_page' => -1, // By default WP_Query will return only 10 posts, to avoid that we need to pass -1
     );
 
+    //check woo-discount-rule plugin disabling the coupons
+    $discountOptions = get_option( 'woo-discount-config-v2', array());
```

Comment:
What if the plugin itself not installed? Is this option available on `get_option` function? @poojarazor

Reply:
This option will not available when plugin not installed , value set as a null value

Reply:
Consider using `[]` over `array()` based on style guide - Chetan please help